README.md

```diff
@@ -260,23 +260,23 @@ $ peg install <cluster-name> <technology>
 ```
 The `technology` tag can be any of the following:
 * alluxio (v1.0.0)
-* cassandra (v3.6)
+* cassandra (v3.7)
 * elasticsearch (v2.3.3)
-* flink (v1.0.0 with hadoop v2.7 and scala v2.10)
+* flink (v1.1.2 with hadoop v2.7 and scala v2.10)
 * hadoop (v2.7.2)
 * hbase (v1.2.1)
 * hive (v1.2.1)
-* kafka (v0.9.0.1 with scala v2.10)
-* kafka-manager (v1.3.0.8)
+* kafka (v0.10.0.0 with scala v2.10)
+* kafka-manager (v1.3.1.8+)
 * kibana (v4.5.1)
 * opscenter
 * pig (v0.15.0)
 * presto (v0.147)
-* redis (v3.0.6)
-* secor (v0.21)
+* redis (v3.2.0)
+* secor (v0.21+)
 * spark (v1.6.1 with hadoop v2.6+)
 * storm (v1.0.1)
-* zeppelin
+* zeppelin (v0.6.1+)
```

Comment:
@rheeluke it seems that it is version 0.7.0 right now. Can you check and update if needed? I checked it here https://github.com/apache/zeppelin/blob/master/pom.xml